server/datastore/mysql/hosts.go

```diff
@@ -981,15 +981,24 @@ func (ds *Datastore) TotalAndUnseenHostsSince(ctx context.Context, daysCount int
 		Total  int `db:"total"`
 		Unseen int `db:"unseen"`
 	}
+
+	// convert daysCount to integer number of seconds for more precision in sql query
+	dur, err := time.ParseDuration(fmt.Sprintf("%dh", daysCount*24))
+	if err != nil {
+		return 0, 0, ctxerr.Wrap(ctx, err, "getting total and unseen host counts")
+	}
+	unseenInterval := int(dur.Seconds())
```

Comment:
Could we do this instead: `unseenInterval := daysCount * 24 * 60 * 60`? Also maybe rename to `unseenSeconds` or something like that.